input/fsh/profiles/group.fsh

```diff
@@ -0,0 +1,78 @@
+ValueSet: GroupTypeSubset
+Id: group-type-subset
+Title: "Person and Practitioner from GroupType Value Set"
+Description: "Valuset to constrain the type element in a Group to just person and practitioner"
+* ^experimental = false
+* ^jurisdiction = $m49.htm#001 "World"
+* ^immutable = true
+* http://hl7.org/fhir/group-type#person
+* http://hl7.org/fhir/group-type#practitioner
+
+
+Extension: MemberFilter
+Id: member-filter
+Title: "Member Filter"
+Description: "Extension to define the population of the group using FHIR REST API parameters"
```

Comment:
Can you include an example in the long description here?

Reply:
Good suggestion - will do!